readme.md

```diff
@@ -20,38 +20,44 @@ __Magnolia__ is a generic macro for automatic materialization of typeclasses for
 Given an ADT such as,
 ```scala
 enum Tree[+T]:
-  case class Branch(left: Tree[T], right: Tree[T])
-  case class Leaf(value: T)
+  case Branch(left: Tree[T], right: Tree[T])
+  case Leaf(value: T)
+
+object Tree:
+  given [T: [X] =>> Print[X]] : Print[Tree[T]] = Print.derived
 ```
-and provided an given instance of `Show[Int]` is in scope, and a Magnolia derivation for the `Show` typeclass
-has been provided, we can automatically derive given typeclass instances of `Show[Tree[Int]]` on-demand, like
+and provided an given instance of `Print[Int]` is in scope, and a Magnolia derivation for the `Print` typeclass
+has been provided, we can automatically derive given typeclass instances of `Print[Tree[Int]]` on-demand, like
 so,
 ```scala
-Branch(Branch(Leaf(1), Leaf(2)), Leaf(3)).show
+Print.derived[Tree[Int]].print(Branch(Branch(Leaf(1), Leaf(2)), Leaf(3)))
```

Comment:
this does not compile after copy-paste - either an import (`Tree.*`) or qualifying the names e.g. `Tree.Branch` is necessary

Reply:
I forgot to put it there... :) Fixed in new commit